app/uk/gov/hmrc/plasticpackagingtax/returns/connectors/TaxReturnsConnector.scala

```diff
@@ -32,17 +34,28 @@ class TaxReturnsConnector @Inject() (
   metrics: Metrics
 )(implicit ec: ExecutionContext) {
 
+  private val logger = Logger(this.getClass)
+
   def find(
     id: String
   )(implicit hc: HeaderCarrier): Future[Either[ServiceError, Option[TaxReturn]]] = {
     import uk.gov.hmrc.http.HttpReads.Implicits.readOptionOfNotFound
     val timer = metrics.defaultRegistry.timer("ppt.returns.find.timer").time()
     httpClient.GET[Option[TaxReturn]](appConfig.pptReturnUrl(id))
       .andThen { case _ => timer.stop() }
-      .map(resp => Right(resp.map(_.toTaxReturn)))
+      .map {
+        resp =>
+          logger.info(s"Found ppt tax returns for id [$id] had response [${toJson(resp)}]")
```

Comment:
Likewise for all the responses in this returns connector. Logging the id will be enough for now.

Reply:
Done

Reply:
This connector hasn't changed - the response needs to be removed from the three log statements. Did it miss being pushed?